app/furniture/marketplace/orders/_order.html.erb

```diff
@@ -26,4 +26,18 @@
   <hr class="col-span-6 mb-2" />
   <div class="col-span-4">Total</div>
   <div class="col-span-2 text-right"><%= humanized_money_with_symbol(order.price_total) %></div>
+
+  <%- if policy(order.events).index? %>
```

Comment:
Do we even need this?

Reply:
Because the user is already authorized at the Order level?

Reply:
More that `Policy#index?` returns "true" by default, so it feels kind of extraneous....